reinforcement_learning/rl_game_server_autopilot/game-env/minecraft-server-image/account.conf

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+  
+account=356566070122
```

Comment:
There is some account information here. This will not work for other accounts?

Reply:
yes, it will work for for other accounts.